### PR TITLE
[CBRD-25477]  Prevent precision and scale from being specified for argument types and return types in SP CREATE statements

### PR DESCRIPTION
**Pull request by ctshim**
http://jira.cubrid.org/browse/CBRD-25477

* Prevent precision and scale from being specified for argument types and return types in SP CREATE statements.
*

### CODE REVIEW

File: msg/en_US.utf8/cubrid.msg [hunk 1]
```diff
@@ -1618,6 +1618,7 @@ $set 7 MSGCAT_SET_PARSER_SYNTAX
 136 Maximum length of a server dbname is %1$d bytes.
 137 Maximum length of a server user is %1$d bytes.
 138 Invalid deduplicate level. Valid level between %1$d and %2$d.
+139 Precision cannot be specified for arguments and return values ​​of procedures and functions.
```
Comment (hyunikn) [inline msg/en_US.utf8/cubrid.msg]:
```suggestion
139 Precision cannot be specified for argument types and return types ​​of stored procedures and functions.
```

File: msg/ko_KR.utf8/cubrid.msg [hunk 2]
```diff
@@ -1618,6 +1618,7 @@ $set 7 MSGCAT_SET_PARSER_SYNTAX
 136 Server의 DBNAME 최대 길이는 %1$d 바이트입니다.
 137 Server의 USER 최대 길이는 %1$d 바이트입니다.
 138 Invalid level. 유효값은 %1$d부터 %2$d까지 입니다.
+139 프로시저와 함수의 아규먼트와 리턴값에는 정밀도를 지정할 수 없습니다.
```
Comment (hyunikn) [inline msg/ko_KR.utf8/cubrid.msg]:
```suggestion
139 저장 프로시저와 함수의 인자타입과 리턴타입에는 정밀도를 지정할 수 없습니다.
```

File: src/parser/csql_grammar.y [hunk 3]
```diff
@@ -21457,150 +21469,157 @@ primitive_type
...
-				  }
+                                dt->type_enum = typ;
+                                dt->info.data_type.precision = l;
+                                switch (typ)
+                                {
+                                case PT_TYPE_CHAR:
+                                case PT_TYPE_VARCHAR:
+                                case PT_TYPE_NCHAR:
+                                case PT_TYPE_VARNCHAR:
+                                        if (pt_check_grammar_charset_collation
```
Comment (hyunikn) [inline src/parser/csql_grammar.y:21580]: indet 가 어떤 식으로든 흐트러졌나 봅니다.

**Review by hyunikn — APPROVED** [review]

File: src/parser/csql_grammar.y (hunk 3 above)
Comment (ctshim) [reply src/parser/csql_grammar.y:21580]: if(is_in_sp_func_type && len)
{
}
else { }  블록으로 감싸지면서 인덴트가 조정되었습니다.

File: msg/ko_KR.utf8/cubrid.msg (hunk 2 above)
Comment (ctshim) [reply msg/ko_KR.utf8/cubrid.msg]: Thank you!

File: msg/en_US.utf8/cubrid.msg (hunk 1 above)
Comment (ctshim) [reply msg/en_US.utf8/cubrid.msg]: Thank you!

**Review by beyondykk9 — APPROVED** [review]

**Review by jongmin-won — APPROVED** [review]

File: msg/ko_KR.utf8/cubrid.msg [hunk 4]
```diff
@@ -1618,6 +1618,7 @@ $set 7 MSGCAT_SET_PARSER_SYNTAX
 136 Server의 DBNAME 최대 길이는 %1$d 바이트입니다.
 137 Server의 USER 최대 길이는 %1$d 바이트입니다.
 138 Invalid level. 유효값은 %1$d부터 %2$d까지 입니다.
+139 저장 프로시저와 함수의 인자타입과 리턴타입에는 정밀도를 지정할 수 없습니다.
```
Comment (hgryoo) [inline msg/ko_KR.utf8/cubrid.msg]:
```suggestion
139 저장 프로시저/함수의 인수 타입과 결과 타입에는 정밀도를 지정할 수 없습니다.
```

**Review by hgryoo — APPROVED** [review]

**Review by vimkim — APPROVED** [review]

File: msg/ko_KR.utf8/cubrid.msg (hunk 4 above)
Comment (ctshim) [reply msg/ko_KR.utf8/cubrid.msg]: Thank you.

**Pull request merged**